schemas/logs/salesforce/login-as.yml

```diff
@@ -0,0 +1,116 @@
+schema: Salesforce.LoginAs
+version: 0
+referenceURL: https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_eventlogfile_loginas.htm
+description: "Login As events contain details about what a Salesforce admin did while logged in as another user."
+parser:
+  csv:
+    delimiter: ","
+    hasHeader: true
+fields:
+  - name: EVENT_TYPE
+    required: true
+    type: string
+    validate:
+      allow: ["LoginAs"]
+    description: The type of event. The value is always LoginAs.
+  - name: TIMESTAMP
+    required: false
+    type: timestamp
+    timeFormat: "%Y%m%d%H%M%S.%f"
```

Comment:
I am not sure `%f` is the correct format here.
From [strftime.org](https://strftime.org/):
 Code  |   Meaning  | Example
-- | -- | --
`%-S` | Second as a decimal number. (Platform specific) | 5
`%f` | Microsecond as a decimal number, zero-padded on the left. | 000000

Note the 'zero-padded on the left' part.
It might be working for the go implementation of the timestamp parser but best check that the format also works in Python as well.

Reply:
It must be zero-padded on the left, in order to be able to parse microseconds expressed as fractions of second, using floating point numbers. What happens is that this format specifier assumes trailing zeroes if the length is less than 6 digits and does not zero-fill from the left.

```python
>>> datetime.datetime.strptime('20210217135741.094500', "%Y%m%d%H%M%S.%f")
datetime.datetime(2021, 2, 17, 13, 57, 41, 94500)
>>> datetime.datetime.strptime('20210217135741.094500', "%Y%m%d%H%M%S.%f").isoformat()
'2021-02-17T13:57:41.094500'
>>> datetime.datetime.strptime('094500', "%f")
datetime.datetime(1900, 1, 1, 0, 0, 0, 94500)
>>> datetime.datetime.strptime('094500', "%f").microsecond
94500
>>> datetime.datetime.strptime('945', "%f").microsecond
945000
```

Reply:
@alxarch is there other feedback or should I proceed with merging maybe?

Reply:
Go ahead, sorry, missed this ping.

Reply:
No worries, thanks for the help!